src/ncp/ncp_base.cpp

```diff
@@ -348,8 +362,13 @@ void NcpBase::HandleReceive(const uint8_t *aBuf, uint16_t aBufLength)
 
     mRxSpinelFrameCounter++;
 
-    // We only support IID zero for now.
-    if (SPINEL_HEADER_GET_IID(header) != 0)
+    mCurCommandIID = SPINEL_HEADER_GET_IID(header);
+
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
+    if (mCurCommandIID > SPINEL_HEADER_IID_MAX)
```

Comment:
Is it possible to set `SPINEL_HEADER_IID_MAX` to `0` when `OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE=0` to simplify this code?

Reply:
@jwhui Thanks for the review comments. I pushed the suggested changes. Please have a look.